rskj-core/src/main/resources/reference.conf

```diff
@@ -299,4 +299,8 @@ prune {
         # in order to avoid forks
         toAvoidForks = 100
     }
+}
+
+cache-size {
```

Comment:
Consider hardcoding this number in code first, so we don't generate lots of config entries that need to be maintained.

Reply:
we addressed this offline